gensim/similarities/docsim.py

```diff
@@ -443,7 +459,14 @@ def save(self, fname=None, *args, **kwargs):
         """
         self.close_shard()
         if fname is None:
-            fname = self.output_prefix
+            if(issubclass(self.__class__, Similarity)):
```

Comment:
Python doesn't use brackets to enclose `if` expressions (see PEP8 for Python code style).